### PR TITLE
Update javadoc for BlockDeviceMappingBuilder

### PR DESCRIPTION
**Pull request by olivergondza**
Update javadoc.

### CODE REVIEW

File: core/src/main/java/org/openstack4j/model/compute/builder/BlockDeviceMappingBuilder.java [hunk 1]
```diff
@@ -37,60 +37,58 @@
 	BlockDeviceMappingBuilder uuid(String id);
 
 	/**
-	 * Either snap or any other value, including a blank string. snap means that the volume was created from a snapshot.
```
Comment (olivergondza) [inline core/src/main/java/org/openstack4j/model/compute/builder/BlockDeviceMappingBuilder.java]: Copy paste error.

File: core/src/main/java/org/openstack4j/model/compute/builder/BlockDeviceMappingBuilder.java [hunk 2]
```diff
@@ -37,60 +37,58 @@
...
 	 */
 	BlockDeviceMappingBuilder volumeSize(Integer size);
-	
+
     /**
      * Used to set the disk_bus, low level detail that some hypervisors
      * (currently only libvirt) may support.
-     * 
-     * @see <a href=
-     *      "https://docs.openstack.org/developer/nova/block_device_mapping.html#block-device-mapping-v2">https://docs.openstack.org/developer/nova/block_device_mapping.html#block-device-mapping-v2</a>
```
Comment (olivergondza) [inline core/src/main/java/org/openstack4j/model/compute/builder/BlockDeviceMappingBuilder.java]: Broken link

File: core/src/main/java/org/openstack4j/model/compute/builder/BlockDeviceMappingBuilder.java [hunk 3]
```diff
@@ -37,60 +37,58 @@
...
-	 * Either snap or any other value, including a blank string. snap means that the volume was created from a snapshot.
+	 * Defines where the volume comes from.
 	 *
 	 * @param type the destination type
 	 * @return BlockDeviceMappingBuilder
 	 */
 	BlockDeviceMappingBuilder destinationType(BDMDestType type);
 
 	/**
-	 * Either snap or any other value, including a blank string. snap means that the volume was created from a snapshot.
```
Comment (olivergondza) [inline core/src/main/java/org/openstack4j/model/compute/builder/BlockDeviceMappingBuilder.java]: Valid values are actually guarded by `BDMSourceType`.

File: core/src/main/java/org/openstack4j/model/compute/builder/BlockDeviceMappingBuilder.java [hunk 4]
```diff
@@ -37,60 +37,58 @@
...
-	
+
     /**
      * Used to set the disk_bus, low level detail that some hypervisors
      * (currently only libvirt) may support.
-     * 
-     * @see <a href=
-     *      "https://docs.openstack.org/developer/nova/block_device_mapping.html#block-device-mapping-v2">https://docs.openstack.org/developer/nova/block_device_mapping.html#block-device-mapping-v2</a>
-     * 
-     * @param disk_bus
```
Comment (olivergondza) [inline core/src/main/java/org/openstack4j/model/compute/builder/BlockDeviceMappingBuilder.java]: Parameter name mismatch.

**Comment by auhlig** [comment]
Hi @olivergondza, are that reminders for yourself or should I do something?

**Comment by olivergondza** [comment]
No, I am explaining what I do and why. I am ok with this as it is.

**Comment by auhlig** [comment]
Ah. Understood. Thanks for this @olivergondza

**Pull request merged**